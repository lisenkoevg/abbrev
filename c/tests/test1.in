apple
orange
orange
apple
peach
peach
#
apple
orange
orange
apple
peach
peach
#
apple
orange
orange
apple
peach
peach
#
