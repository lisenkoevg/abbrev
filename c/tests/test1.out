apple
orange
orange
apple
peach
peach

apple
apple
orange
orange
peach
peach

apple
orange
peach
